docs/installation/linux.md

```diff
@@ -106,6 +107,14 @@ sudo yum remove powershell
 
 [CentOS 7]: https://www.centos.org/download/
 
+Arch Linux
+==========
+Powershell is available from the Arch Linux User Repository (AUR) as a [release](https://aur.archlinux.org/packages/powershell/) or the [latest development build](https://aur.archlinux.org/packages/powershell-git).
+
+For more information on installing packages from the AUR, see the [Arch Linux wiki](https://wiki.archlinux.org/index.php/Arch_User_Repository#Installing_packages).
+
+[Arch Linux]: https://www.archlinux.org/download/
+
```

Comment:
Please also add a note that the AUR repos for PowerShell are community maintained, so that these docs don't imply official support.